src/main/resources/assets/jei/lang/de_DE.lang

```diff
@@ -1,64 +1,88 @@
 # Tooltips
-jei.tooltip.config=Konfiguration
+jei.tooltip.config=JEI-Konfiguration
+jei.tooltip.show.recipes=Zeige Rezepte
 jei.tooltip.show.all.recipes=Zeige alle Rezepte
 jei.tooltip.delete.item=Klicke zum Löschen
 jei.tooltip.liquid.amount.with.capacity=%,d / %,d mB
 jei.tooltip.liquid.amount=%,d mB
+jei.tooltip.transfer=Bewege Gegenstände
+jei.tooltip.recipe.ore.dict=Akzeptiert: %s
+jei.tooltip.item.colors=Farben: %s
+jei.tooltip.shapeless.recipe=Formloses Rezept
+jei.tooltip.cheat.mode=Cheat-Modus
 
 # Error Tooltips
-jei.tooltip.error.recipe.transfer.missing=Fehlende Items
-jei.tooltip.error.recipe.transfer.inventory.full=Inventar ist voll
+jei.tooltip.error.recipe.transfer.missing=Fehlende Gegenstände
```

Comment:
Well, hello... :-)
Yeah, we can use "Item" in German, I have no "anti-item-ideology" or something like that. But I think it's important to not let german words disappear just to make it consistent at any cost. Synonyms prove that people are capable of using more than one word to describe a thing, so why do you want to erase the german word "Gegenstand" from computer games as if it's something that doesn't belong there?
I suggested the "hybrid mode": e.g. call it "Item-Cheat" and in the tooltip, explain it with "Gibt den Gegenstand, statt das Rezept zu zeigen." In that case, people who don't speak English at all can learn that an "Item" is a "Gegenstand", because they know the latter, and people who already know simply don't bother. I just don't understand why you are bothered by this.
Following you logic you should also erase all the other synonyms, as people could be confused by them too. I don't think that people are that stupid, and if it seems so, e.g. "pipe" vs. "tube", than PLEASE don't let the stupidity win by giving up, but instead teach them so they can learn. Isn't that a better way?